src/components/layout/footer.tsx

```
@@ -0,0 +1,74 @@
+}
+
+export function Footer() {
+    const [commit] = createResource(fetchLatestCommit)
+
+    const version = () => {
+        if (!commit.latest) return 'v1.0.1'
+        const patch = BASE_VERSION.patch + 1
+        return `${BASE_VERSION.major}.${BASE_VERSION.minor}.${patch}`
+    }
+
+    return (
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)